src/core/gui/dialog/backgroundSelect/PdfPagesDialog.cpp

```diff
@@ -44,7 +41,7 @@ PdfPagesDialog::PdfPagesDialog(GladeSearchpath* gladeSearchPath, Document* doc,
         }
     }
 
-    updateOkButton();
+    gtk_widget_set_sensitive(get("buttonOk"), false);
```

Comment:
By "default", if a page exists, it will still be enabled with the first page selected, as before. I do think it would be more logical to have no page selected and the button inactive at the beginning, but that would be a change in behavior. It'd be just 3 more line to remove here: https://github.com/xournalpp/xournalpp/blob/c2b7f6553ec8f7ad053bcdd3895f394b25549bea/src/core/gui/dialog/backgroundSelect/BackgroundSelectDialogBase.cpp#L88-L90